tests/jballerina-unit-test/src/test/resources/test-src/annotations/annot_access.bal

```diff
@@ -359,6 +359,41 @@ type MedicalNeed record {
 
 };
 
+public type AnnotationRecord record {|
+    string summary?;
+    Examples examples?;
+|};
+
+public type Examples record {|
+    map<ExampleItem> response;
+|};
+
+public type ExampleItem record {
+    map<string> headers?;
+};
+
+const annotation AnnotationRecord annot on type;
+
+@annot {
+    examples: {
+        response: {}
+    }
+}
+type Employee record {|
+    int id;
+    string name;
+|};
+
+Employee employee = {id: 1, name: "chirans"};
```

Comment:
Why does this have to be at module-level?

Reply:
move the logic inside a function